api/HTMLCollection.json

```diff
@@ -127,10 +127,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤4"
```

Comment:
Let's just set it to 1. The interface makes no sense without the length property, and the rest of the data is 1.